src/store/middlewares/recordingMiddleware.tsx

```diff
@@ -68,6 +68,8 @@ const createRecordingMiddleware = ({
 		getState: RootState
 	}) =>
 		(next) => async (action) => {
+			mimeType = getState().settings.preferredRecorderMimeType;
```

Comment:
Please move this lookup into the if statement below, or this lookup will be done on all dispatches to the store.